### ISSUE TITLE
Add multi-allele genotype 

### ISSUE THREAD

**Issue by ValWood** [pombase/pombase-chado#1158]
We don't have a way to capture
cdc1+(3 copies)	b5089e77a1ba9a37
  cdc1+(1 copy)	b5089e77a1ba9a37
  cdm1+(1 copy)	b5089e77a1ba9a37
  cdm1+(3 copies)	b5089e77a1ba9a37
  cdc27+(1 copy)	b5089e77a1ba9a37
  cdc27+(3 copies)	b5089e77a1ba9a37
  cdc6+(1 copy)	b5089e77a1ba9a37
  cdc6+(3 copies)	b5089e77a1ba9a37

because we can't add multiple copies of a single gene.
Perhaps we can add this to the database directly, bypassing Canto? In a similar way that you did the diploid genotypes for @PCarme  recently?

I'm not sure it is worth making a Canto change because this is the only example I'm aware of.

**Comment by kimrutherford** [comment]
Just so I'm clear, do you want to be able to make genotypes like this?:

![image](https://github.com/pombase/pombase-chado/assets/90474/93a28742-4b64-4b37-b4f5-530c18a25050)

**Comment by ValWood** [comment]
Yes exactly. They integrated multiple copies to increase expression level.

**Comment by kimrutherford** [comment]
We could allow this for admin users only?  Currently the Finish button is greyed-out but it would be easy to enable it out for admins.

> Perhaps we can add this to the database directly, bypassing Canto?

You'll need them in Canto to annotate them wouldn't you?

**Comment by ValWood** [comment]
I was envisaging we could just load these into chado in some way and bypass canto, but for this I realise now I need to supply the annotations too, not just the genotypes!

It's such an edge case it might not be worth altering Canto. But if it is a simple change yes, admin only.

**Comment by kimrutherford** [comment]
> But if it is a simple change yes, admin only.

It was a 2-line change so it's done now and deployed in the main Canto.

Let me know how it goes.

**Comment by ValWood** [comment]
I was not able to create multiple identical copies....

**Comment by ValWood** [comment]
I could still not do it. We can have a look tomorrow.
v

**Comment by kimrutherford** [comment]
Perhaps we can chat about this on Zoom today?  I just need a quick demo to work out what the problem is.
